docs/build/_index.md

```diff
@@ -10,9 +10,23 @@ no_list: true
 menuindent: true
 ---
 
-Viam uses a composable system of building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} which you can combine according to your specific needs.
+Viam uses a composable system of open source building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} which you can combine according to your specific needs.
```

Comment:
Super nit, but would not mention open source since technically modules can be closed source, I believe

Reply:
```suggestion
Viam uses a composable system of building blocks called {{< glossary_tooltip term_id="resource" text="resources" >}} which you can combine according to your specific needs.
```